eos-tech-support/eos-download-image

```diff
@@ -284,6 +284,7 @@ class EosDownloadImage(object):
         dest = os.path.join(self.args.outdir, filename)
         log("Fetching Windows Installer")
         self.download(url, dest)
+        print(dest)
```

Comment:
Could you add a comment on why this is needed? Either in the function docstring or just a normal comment above the print?